server/channels/store/searchtest/file_info_layer.go

```diff
@@ -127,7 +127,7 @@ var searchFileInfoStoreTests = []searchTest{
 	{
 		Name: "Should not support search with preceding wildcards",
 		Fn:   testFileInfoNotSupportPrecedingWildcards,
-		Tags: []string{EngineAll},
+		Tags: []string{EnginePostgres, EngineMySql, EngineElasticSearch},
```

Comment:
As the preceding wildcard is implicitly allowed on `BleveSearch`, I removed the `EngineBleve` tag from the tags.